geode-dunit/src/main/java/org/apache/geode/test/junit/rules/LocatorStarterRule.java

```diff
@@ -58,6 +58,11 @@
 public class LocatorStarterRule extends MemberStarterRule<LocatorStarterRule> implements Locator {
   private transient InternalLocator locator;
 
+  public LocatorStarterRule() {
+    // in test environment, enable management request/response logging
+    systemProperties.setProperty("geode.management.request.logging", "true");
```

Comment:
Would calling `withSystemProperty("geode.management.request.logging", "true");` be better? Its implementation just calls "put" instead of "setProperty" but I do see it is used by some other tests.